Python/DataStructures/Stack/stack.py

```diff
@@ -1,8 +1,9 @@
+
```

Comment:
Mentioned in the description but figured I'd repeat here for clarity, `Data-Structures` isn't a valid python package name and as a result meant that the files couldn't be used for imports.